src/Includes/Actions.php

```diff
@@ -61,4 +62,57 @@ public function register_assets() {
 			wp_add_inline_script( 'plausible-analytics', 'plausible("404",{ props: { path: document.location.pathname } });' );
 		}
 	}
+
+	/**
+	 * Create admin bar nodes.
+	 *
+	 * @since  1.2.0
+	 * @access public
+	 *
+	 * @return void
+	 */
+
+	public function admin_bar_node( $admin_bar ) {
+		// Add main admin bar node
+		$args = [
+			'id'    => 'plausible-admin-bar',
+			'title' => 'Plausible Analytics',
+		];
+		$admin_bar->add_node( $args );
+
+		// Add sub menu items
+
+		// Add link to view all stats
+		$args   = [];
+		$args[] = [
+			'id'     => 'view-analytics',
+			'title'  => 'View Analytics',
+			'href'   => admin_url( 'index.php?page=plausible_analytics_statistics' ),
+			'parent' => 'plausible-admin-bar',
+		];
+
+		// Add link to individual page stats
+		if ( ! is_admin() ) {
+			global $post;
+			$page_url  = admin_url( 'index.php?page=plausible_analytics_statistics' );
+			$page_url .= is_home() ? '' : '&page-url=' . trailingslashit( urlencode( '/' . $post->post_name ) );
+			$args[]    = [
+				'id'     => 'view-page-analytics',
+				'title'  => 'View Page Analytics',
+				'href'   => $page_url,
```

Comment:
Can you please use `sprintf` along with `add_query_arg` to simplify `$page_url` and directly attach it with `href` parameter?